apps/meteor/client/views/room/contextualBar/RoomMembers/AddUsers/ValidateMatrixInvitedUsers/ValidateMatrixIdModal.tsx

```diff
@@ -0,0 +1,63 @@
+import { Modal, Button, Box, Icon } from '@rocket.chat/fuselage';
+import { useTranslation } from '@rocket.chat/ui-contexts';
+import type { ReactElement } from 'react';
+import React from 'react';
+
+type ValidateMatrixIdModalProps = {
+	_matrixIdVerifiedStatus: Map<string, string>;
+	onClose: () => void;
+	onConfirm?: () => void;
+};
+
+const ValidateMatrixIdModal = ({ onClose, onConfirm, _matrixIdVerifiedStatus }: ValidateMatrixIdModalProps): ReactElement => {
+	const t = useTranslation();
+
+	const verificationStatusAsIcon = (verificationStatus: string) => {
```

Comment:
You can move this function out of the Component scope, since it doesnt use any variables defined within it.

I recommend making this function return only the Icon `name` string property instead of returning the element.